<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIBankSimeasymode</class>
 <widget class="QWidget" name="UIBankSimeasymode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="navWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1881</width>
     <height>1041</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Nav">
    <widget class="QGroupBox" name="groupBoxNav">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>19</y>
       <width>1841</width>
       <height>1001</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="buttonWithdraw">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>150</y>
        <width>600</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Withdraw</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonLogOut">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>410</y>
        <width>600</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBalance">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>150</y>
        <width>600</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-color: rgb(0, 255, 0);</string>
      </property>
      <property name="text">
       <string>Account balance</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelATM">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>10</y>
        <width>500</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>52</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ATM</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonTransactions">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>410</y>
        <width>600</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Guide</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Balance">
    <widget class="QGroupBox" name="groupBoxBalance">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>1841</width>
       <height>1001</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="labelTitleAccountBalance">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>0</y>
        <width>791</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>52</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Account Balance</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelTitleSaldo">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>120</y>
        <width>571</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>80</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Balance:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBackBalance">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>660</y>
        <width>561</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>52</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSaldo">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>1241</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>80</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>00.00€</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Withdraw">
    <widget class="QGroupBox" name="groupBoxWithdraw">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>19</y>
       <width>1841</width>
       <height>1001</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="button240">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>520</y>
        <width>400</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>52</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>240€</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>0</y>
        <width>471</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>52</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Withdraw</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="button140">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>350</y>
        <width>400</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>52</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>140€</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>70</y>
        <width>471</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SELECT AMOUNT:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="button20">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>180</y>
        <width>400</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>52</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>20€</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button60">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>520</y>
        <width>400</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>52</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>60€</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBackWithdraw">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>700</y>
        <width>400</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>52</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button40">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>350</y>
        <width>400</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>52</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>40€</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button90">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>180</y>
        <width>400</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>52</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>90€</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="withdrawCustom">
    <widget class="QGroupBox" name="groupBoxWithdrawCustom">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>1841</width>
       <height>1001</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="buttonNum1">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>260</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonNum9">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>560</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonNum2">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>260</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonNum6">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>410</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonClear">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>710</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CLEAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonNum5">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>410</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonNum0">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>710</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>120</y>
        <width>441</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonEnter">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>710</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ENTER</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonNum8">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>560</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBack">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>710</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonNum4">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>410</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonNum7">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>560</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonNum3">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>260</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTitleCustomAmount">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>40</y>
        <width>431</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Withdraw custom amount:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Transactions">
    <widget class="QGroupBox" name="groupBoxTransactions">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>19</y>
       <width>1841</width>
       <height>1001</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="LabelTitleLastLogin">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>130</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Last Login:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPageNumber">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>690</y>
        <width>41</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelLastLogin">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>130</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>YYYY-MM-DD HH:MM:SS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="previousPageButton">
      <property name="geometry">
       <rect>
        <x>742</x>
        <y>690</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;Previous Page</string>
      </property>
     </widget>
     <widget class="QWidget" name="widgetTransaction" native="true">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>210</y>
        <width>691</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBackTransactions">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>780</y>
        <width>301</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelBalance">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>170</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>00.00 €</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelTitleBalance">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>160</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Balance:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>40</y>
        <width>721</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Transactions</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="nextPageButton">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>690</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Next Page&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="withdrawDone">
    <widget class="QGroupBox" name="groupBoxWithdrawDone">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>1841</width>
       <height>1001</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="labelEuroTitle">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>200</y>
        <width>531</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Euros withdrawn</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelEuroAmount">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>200</y>
        <width>411</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>00.00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>320</y>
        <width>381</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Logging out</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
